creator-node/src/serviceRegistry.js

```diff
@@ -26,27 +26,26 @@ class ServiceRegistry {
     this.ipfsLatest = ipfsLatest
     this.blacklistManager = BlacklistManager
     this.monitoringQueue = new MonitoringQueue()
+    this.nodeConfig = config
 
-    // this.audiusLibs isn't initialized until
-    // `initServices` is called.
-    this.audiusLibs = null
-    // this.snapbackSM isn't initialized until 'initServices'
+    // below properties aren't initialized until 'initServices' is called
+    this.libs = null
```

Comment:
this is just a bugfix - i think this property was accidentally defined as `this.audiusLibs` before but is referenced as `this.libs` below and elsewhere